index.md

```diff
@@ -665,97 +665,133 @@ they keep track of what was changed in a file when and by whom,
-
-A good version control system should allow a researcher to:
-
-1.  understand and review the entire development history of a file,
-    including identifying when a particular change was made and who made it;
-2.  locate the place in the history where a particular change was introduced,
-    easily and quickly;
-3.  evaluate an individual change to a file so that its effects can be understood
-    and, if necessary, undone;
-4.  manage concurrent changes from many different collaborators; and
-5.  make projects discoverable through portals like GitHub.
-
 We believe that the best tools for providing these capabilities are
 version control systems that are widely used in software development,
 such as Git, Mercurial, and Subversion
 (which all provide the same basic functionality).
-We therefore recommend that researchers use one of these.
-
 However,
-it is important to distinguish between the concept of version control and particular implementations.
-In particular,
-we recognize that many newcomers find these tools difficult to learn
-and that the benefits of version control tools may only become apparent
+many newcomers find these tools difficult to learn,
+and their benefits often only become apparent
 on projects that are larger and more complex
 than the "solo researcher" projects common in many academic fields.
 
-We therefore also suggest a second, manual approach for managing changes
+For those not yet ready to adopt version control,
+we recommend a systematic manual approach for managing changes
 that can be appropriate for small projects
 with one or a few investigators working closely together.
 
-**FIXME: Justin, I've never seen this CHANGELOG approach implemented -
-can you please point us at a project where you or one of your colleagues do this?**
+Whatever approach is adopted should help with the following:
+
+1.  It should aid *reproducibility*
+    by allowing you to reference or retrieve a specific version of the entire project.
+    This is valuable for your future self (when you finally get the reviews back for your paper),
+    for your lab-mates and collaborators (in case you leave the project),
+    and for reviewers, editors, and others who want to convince themselves of the conclusions in your published research.
+    It also helps your future self come back to a project after a long field season
+    and *not* have to spend days remembering what you did last
+    or figuring out what your collaborators have changed while you have been gone.
+2.  It should aid *fixability*:
+    having access to every version of code, figures, and data helps you figure out
+    why Figure 4 looks different now than it did last week.
+3.  It should support *sharing and collaboration*,
+    particularly by managing the process of merging independent changes made by different people,
+    and distributing those changes back to everyone in a controlled, traceable way.
+
+The "by hand" approach has three steps.
```

Comment:
I'd more clearly delineate the two approaches (perhaps even with another subheading but that may be overkill). E.g.

'Our first suggested, "by hand" approach has three steps.'